docs/components/movement-sensor/viam-visual-odometry.md

```diff
@@ -78,8 +79,8 @@ To configure the `viam-visual-odometry` module on your robot, follow the instruc
    }
    ```
 
-   Provide the same camera name as you used in step 3.
-   See the [Attributes](#attributes) section for more information on the other attributes.
+Provide the same camera name as you used in step 4.
```

Comment:
```suggestion
Provide the camera name you used in step 4.
```

nit: either `you used` or `as used`, not both.

Reply:
Good catch--I was not even reviewing the text, just fixing the number.